### ISSUE TITLE
Uncaught ReferenceError: loadBimServerApi is not defined

### ISSUE THREAD

**Issue by carlosips** [opensourceBIM/BIMsurfer-before2019#130]
In example1.js file, in connect function, there is a loadBimServerApi function call, but isn't defined. The file where this function is defined has been removed since last commit. How can I connect to bimserver? Besides, Are there any way to use only bimsurfer (without bimserver) in my own project?

**Comment by rechappoux** [comment]
Did you find the solution?

**Comment by carlosips** [comment]
No. Finally I have used [IfcConvert](http://ifcopenshell.org/ifcconvert.html) and [Three.js](http://threejs.org/).

**Comment by harrio** [comment]
The current way to load the API seems to be here: https://github.com/opensourceBIM/BIMserver/wiki/JavaScriptClient

At least I managed to load the API and log into the server. It did require a bit of debugging as the information given on the page is quite sketchy or plain wrong like much of BIMServer's documentation, unfortunately.

**Comment by JanBurp** [comment]
Same issue here :-(
